cypress/integration/brewery/TableParameters-files_part4.spec.js

```diff
@@ -45,7 +45,7 @@ describe('Table parameters files standard operations part 4', () => {
     }
   });
 
-  it('can create a scenario with default table parameter', () => {
+  it('can create a scenario with default table parameter, edit it, discard, re-edit, save, re-edit, re-save', () => {
```

Comment:
Do we really need to add  the steps "edit it, discard," to reproduce the bug?
If these steps are already checked in other tests, we should not add them in this one.